packages/ng-form/src/lib/components/form/form.component.spec.ts

```diff
@@ -0,0 +1,192 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing'
```

Comment:
In general, it would be cool to use [angular-testing-library](https://testing-library.com/docs/angular-testing-library/intro/). Maybe I'll rewrite them myself using that library, which IMO is far better than `TestBed`

Reply:
I tried it... but it seemed that for this case it was complicating the tests. I spent 30 mins without getting anything good. Maybe it's my fault and I was doing the tests in the wrong way with the library...

Reply:
Maybe its just better `TestBed` 🤷🏻‍♂️ I assumed `testing-library` was better because of the experience I had with both Cypress and React, but maybe that's not the case with Angular. Thanks for trying it out!